coresimd/aarch64/neon.rs

```diff
@@ -138,6 +138,27 @@ extern "C" {
     #[link_name = "llvm.aarch64.neon.fminv.f64.v2f64"]
     fn vminvq_f64_(a: float64x2_t) -> f64;
 
+    #[link_name = "llvm.arm.neon.sha1h"]
```

Comment:
llvm.arm.neon hints that they do belong in the `coresimd/arm/neon.rs` module

Reply:
OK, good! I wasn't sure if `crypto.rs` would make more sense.